src/cnaas_nms/tools/jinja_filters.py

```diff
@@ -0,0 +1,41 @@
+import ipaddress
+
+
+def increment_ip(ip_string, increment=1):
+    """Increment an IP address by a given value. Default increment value is 1.
+    Args:
+        ip_string: IP address string. Can be plain or with numeric /prefix
+        increment: Optional increment step, defaults to 1
+    Returns:
+        String with the incremented IP address, with optional numeric prefix
+    """
+    # plain IP
+    try:
+        ip = ipaddress.ip_address(ip_string)
+        return format(ip + increment)
+    # IP with prefix
+    except ValueError:
+        pass  # do not chain exceptions, this is expected behaviour
```

Comment:
I think it's easier to read/understand the logic if the "return increment_if" is right here in the except clause where you check for the error so you can see that they go together? Or if that's not a good idea then instead of a try/catch statement do "if '/' in ip_string: then ip_address else: ip_interface"
If we can handle both cases (ip and interface) I don't think there's a need to expose the increment_if function at all to users?